manifests/cli.pp

```diff
@@ -49,12 +53,14 @@
     $foreman_url_real = pick($foreman_url, $::foreman::foreman_url)
     $username_real    = pick($username, $::foreman::admin_username)
     $password_real    = pick($password, $::foreman::admin_password)
+    $ssl_ca_file_real = pick($ssl_ca_file, $::foreman::server_ssl_ca)
   } else {
     $foreman_url_real = $foreman_url
     $username_real    = $username
     $password_real    = $password
+    $ssl_ca_file_real = $ssl_ca_file
   }
-  validate_string($foreman_url_real, $username_real, $password_real)
+  validate_string($foreman_url_real, $username_real, $password_real, $ssl_ca_file_real)
```

Comment:
This may not be needed but the documentation states it should be an absolute path and here it's only validated as a string.